scripts/node/refreshDatabase.js

```diff
@@ -31,11 +36,12 @@ class RefreshDatabaseScript extends Script {
     this.logInfo('Refreshing the tupaia database...');
 
     this.verifyPsql();
-    this.execDbCommand('DROP DATABASE IF EXISTS tupaia');
-    this.execDbCommand('CREATE DATABASE tupaia WITH OWNER tupaia');
-    this.execDbCommand('CREATE EXTENSION postgis', { db: 'tupaia' });
+    const dbName = this.args.database || 'tupaia';
+    this.execDbCommand(`DROP DATABASE IF EXISTS ${dbName}`);
+    this.execDbCommand(`CREATE DATABASE ${dbName} WITH OWNER tupaia`);
+    this.execDbCommand('CREATE EXTENSION postgis', { db: dbName });
     this.execDbCommand('ALTER USER tupaia WITH SUPERUSER');
-    this.exec(`psql -U tupaia -f "${this.args.dumpPath}"`); // Relative to the repo root
+    this.exec(`psql -U tupaia -d ${dbName} -f "${this.args.dumpPath}"`); // Relative to the repo root
```

Comment:
In some ways I feel it may be better for us to just add a `.env` file to the root directory with our standard db env vars (`DB_NAME`, `DB_USER`, etc.), thoughts on that? But also happy with this as well 👍

Reply:
I also thought that we could use the .env file from the database package, since most probably the same value will be used here and there. But then I wasn't sure if this script should depend in any information from that package - although now that you mentioned, perhaps this script should live in the `database` package in the first place!

Reply:
How do you feel about the following?
* Move the `refreshDatabase.js`/`dumpDb.sh` scripts under the `database` package
* Make the first read the `.env` files from there to determine the default db name (instead of hard coding them to `tupaia`)
* Maintain support for the existing `yarn` scripts in the root level - they can just call the `database` scripts

Reply:
That sounds great!